### DISCUSSION TITLE
Dev Doc Feedback

### DISCUSSION THREAD

**Discussion by danielkestrel** [aptos-labs/aptos-developer-discussions#60] [Dev Doc and Tutorial Feedback]
Can we create/new repo for simple guides, merge with the official Aptos doc?

**Comment by 0x-j** [comment]
Hi can you be more specific on what kind of guide you want? More tutorials in aptos.dev? We are working on a new site [aptos learn](https://learn.aptoslabs.com/) that has some tutorials, and we are still in the process of adding more. Let us know if there's any specific example or tutorial you want to see!

**Reply by danielkestrel** [reply]
Thank you. I was thinking of the possibility of assisting, if it's open source. For example, I had an ish implementing wallet collection some weeks ago. It took a while until Greg stepped in. So i was thinking I could help others with same knowledge.
